### PR TITLE
Validate output with validate_script_output instead of needling

### PR DESCRIPTION
**Pull request by dgutu-suse**
Based on https://progress.opensuse.org/issues/9660

### CODE REVIEW

**Comment by dgutu-suse** [comment]
A working example on Leap - http://crocodile.qa.suse.cz/tests/963

**Comment by okurz** [comment]
I like your approach, proper issue referencing and showing working local example. I would like others to show the same :-)

**Comment by coolo** [comment]
but you need to rebase it seems

File: tests/console/a2ps.pm [hunk 1]
```diff
@@ -1,16 +1,18 @@
 use strict;
 use base "consoletest";
 use testapi;
-
-# a2ps is available by default only in openSUSE*
+use ttylogin;
```
Comment (coolo) [inline tests/console/a2ps.pm]: this module no longer exist - and you don't use it anyway

Comment (dgutu-suse) [inline tests/console/a2ps.pm]: Yes, I don't and didn't.

**Pull request closed without merging**